clmm/gcdata.py

```diff
@@ -65,6 +71,16 @@ def __init__(self, *args, **kwargs):
         APtable.__init__(self, *args, **kwargs)
         metakwargs = kwargs["meta"] if "meta" in kwargs else {}
         metakwargs = {} if metakwargs is None else metakwargs
+        if (
+            "coordinate_system" not in metakwargs
```

Comment:
what is the application of `copy_indices` and `copy` here?

Reply:
@caioolivv I think adding these changes on `__getitem__` is the better approach. As you mentioned, `cosmo` was already being lost when a slice was made. What was the problem with pylint? Depending on the error we can ignore it